robottelo/decorators/host.py

```diff
@@ -0,0 +1,120 @@
+import logging
+from functools import wraps
+
+import re
+import six
+import unittest2
+
+from robottelo import ssh
+from robottelo.config import settings
```

Comment:
Regarding importing order the coding style suggests:

1. - standard library imports
2. - related third party imports
3. - local application/library specific imports

So following that general rule should be
```python
import logging
import re
from functools import wraps

import six
import unittest2
from robottelo import ssh
from robottelo.config import settings
```

But if you look the other modules in robottelo you will see that in Robottelo the style guide is:

```python
# for each imported group the order of imports should be:
import <package>.<module> style lines in alphabetical order
from <package>.<module> import <symbol> style in alphabetical order
```

So maybe the best to follow the project style is:

```python
import logging
import re
import six
import unittest2

from functools import wraps
from robottelo import ssh
from robottelo.config import settings
```